submit-api/src/submit_api/services/submission/__init__.py

```diff
@@ -60,4 +68,10 @@ def edit_submission_form(cls, submission_id, request):
         submission = cls.get_submission_by_id_and_validate_edit(submission_id)
         submission.submitted_form.submission_json = request.get('data')
         submission.submitted_form.save()
+        item_id = request.get("item_id")
+        status = request.get("status")
+        # Update the item with the status from the request
+        if status is not None:
+            update_data = {"status": status}
+            ItemService.update_item(item_id, update_data)
```

Comment:
not a good place to add this  code, this function is to validate not to do update operations